releases/CHANGELOG.md

```diff
@@ -4,6 +4,36 @@ All notable changes to `grf` will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.0] - 2021-?-?
```

Comment:
Let's update this to 2.0.0 here and in tarball name in case we need a patch release (like we did for 1.0.0 :))